app/src/main/java/com/example/sopt30/SignInActivity.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.sopt30
+
+import android.app.Activity
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import com.example.sopt30.databinding.ActivitySignInBinding
+import kotlin.math.sign
+
+class SignInActivity : AppCompatActivity() {
+    private lateinit var binding : ActivitySignInBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivitySignInBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val homeIntent = Intent(this, HomeActivity::class.java)
+        val signUpIntent = Intent(this, SignUpActivity::class.java)
+
+        binding.btnLogin.setOnClickListener{
+            val et_id = binding.etId.getText().toString();
+            val et_pw = binding.etPw.getText().toString();
```

Comment:
```suggestion
            val etId = binding.etId.text.toString();
            val etPw = binding.etPw.text.toString();
```
- 코틀린 스타일 가이드에 따라, 변수명은 lowerCamelCase로 작성하는 게 좋을 것 같아요 :)
- getter인 getText()말고 프로퍼티인 `text`를 활용하는 게 보다 간결하게 표현될 것 같아요 :)